Source/UI/NewUI/BillingForm/BillingFormCell.swift

```diff
@@ -0,0 +1,51 @@
+import Foundation
+
+@frozen public enum BillingFormCell {
+    case fullName(CKOTextFieldCellStyle?)
+    case addressLine1(CKOTextFieldCellStyle?)
+    case addressLine2(CKOTextFieldCellStyle?)
+    case city(CKOTextFieldCellStyle?)
+    case state(CKOTextFieldCellStyle?)
+    case postcode(CKOTextFieldCellStyle?)
+    case country(CKOTextFieldCellStyle?)
+    case phoneNumber(CKOTextFieldCellStyle?)
+    
+    internal var validator: Validator {
+        switch self {
+        case .fullName: return FullNameValidator()
+        case .addressLine1: return AddressLine1Validator()
+        case .addressLine2: return AddressLine2Validator()
+        case .city: return CityValidator()
+        case .state: return StateValidator()
+        case .postcode: return PostcodeValidator()
+        case .country: return CountryValidator()
+        case .phoneNumber: return PhoneNumberValidator()
+        }
+    }
+    
+    internal var style: CKOTextFieldCellStyle? {
+        switch self {
+        case .fullName(let style): return style
+        case .addressLine1(let style): return style
+        case .addressLine2(let style): return style
+        case .city(let style): return style
+        case .state(let style): return style
+        case .postcode(let style): return style
+        case .country(let style): return style
+        case .phoneNumber(let style): return style
+        }
+    }
+    
+    var hash: Int {
```

Comment:
the name `hash` here is quite confusing. it's more of an index or order on the billing details form.